code/game/objects/items/stacks/money.dm

```diff
@@ -93,6 +93,33 @@
 	amount = 1
 	value = 1
 
+/obj/item/stack/money/rubles/New()
+	if(amount < 50)
+		icon_state = "ruble"
+	if(amount >= 50)
+		icon_state = "ruble50"
+	if(amount >= 100)
+		icon_state = "ruble100"
+	if(amount >= 250)
+		icon_state = "ruble250"
+	if(amount == 500)
+		icon_state = "ruble500"
+	update_icon()
+	return ..()
+
+/obj/item/stack/money/rubles/update_icon()
```

Comment:
switch(amount)
if(0 to 49)
     icon_state = "ruble"
 if(50 to 99)
     icon_state = "ruble50"
 if(100 to 249)
     icon_state = "ruble100"
 if(250 to INFINITY)
       icon_state = "ruble500"